TNE/src/net/tnemc/core/common/currency/ItemCalculations.java

```diff
@@ -132,13 +133,21 @@ public static void setItems(UUID account, TNECurrency currency, BigDecimal amoun
       boolean add = (consolidate) || amount.compareTo(old) >= 0;
       if(remove) add = false;
 
-      if(consolidate) split = (amount.toPlainString() + (amount.toPlainString().contains(".")? "" : ".00")).split("\\.");
+      if(consolidate) split = (amount.toPlainString() + (amount.toPlainString().contains(".")? "" : "0.00")).split("\\.");
```

Comment:
This needs to be `.00`.

Reply:
Are you saying that it needs to be .00 instead of the 0.00 you added?

Reply:
Yup. I'll get around to it. Been busy lately.